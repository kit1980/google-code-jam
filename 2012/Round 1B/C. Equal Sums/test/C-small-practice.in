10
20 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
20 120 266 858 1243 1657 1771 2328 2490 2665 2894 3117 4210 4454 4943 5690 6170 7048 7125 9512 9600
20 2694 380 2502 1276 2100 1496 2998 1798 4978 22671 3502 3430 3732 4804 2438 4468 3530 4224 60 50410
20 2494 3418 3062 4536 49426 432 4478 2710 3676 4692 3710 568 3344 440 4328 2008 3690 11201 576 1264
20 22562 36483 27942 94244 30561 21479 60833 63820 72302 1414 32563 50437 67469 34511 44352 94957 74836 72337 91554 83614
20 53608 86390 71480 86249 53027 10813 72946 97047 99682 34026 4069 69289 53720 1651 31507 82584 66916 13026 98893 91949
20 72600 74500 27934 18093 95685 3267 94618 63734 50236 86362 68291 79364 46238 44219 97928 62377 14095 98790 74962 64772
20 84455 33360 6590 50015 64452 46841 35280 41262 80789 24756 73927 98298 38012 49537 99440 57908 27402 26909 23335 64317
20 519 4255 4085 3303 3703 2757 2492 3590 1738 1592 3572 4911 53299 2351 4228 1237 3946 206 3359 1455
20 75799 6551 34257 14031 42965 79428 59813 79687 85386 55401 95129 37596 6787 76097 76330 71680 80779 68085 52241 64784
