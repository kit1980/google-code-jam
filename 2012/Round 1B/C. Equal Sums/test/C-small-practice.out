Case #1:
1 19 
20 
Case #2:
120 1243 1657 1771 2490 2665 2894 5690 
266 4210 4454 9600 
Case #3:
2694 380 
1276 1798 
Case #4:
2494 3062 4536 432 4478 3710 3344 440 2008 3690 
3062 4536 2710 3676 568 3344 440 4328 3690 576 1264 
Case #5:
22562 30561 21479 63820 72302 1414 32563 50437 67469 34511 94957 74836 
94244 60833 63820 72302 67469 44352 72337 91554 
Case #6:
53608 86390 10813 97047 1651 31507 98893 91949 
71480 10813 72946 97047 34026 4069 82584 98893 
Case #7:
72600 95685 50236 86362 68291 97928 14095 98790 74962 64772 
74500 95685 94618 50236 86362 79364 46238 97928 98790 
Case #8:
84455 50015 64452 46841 80789 73927 98298 57908 27402 64317 
33360 6590 50015 64452 46841 98298 49537 99440 57908 27402 26909 23335 64317 
Case #9:
519 4255 4085 3303 3703 2757 2492 3590 1738 1592 3572 4911 2351 4228 1237 3946 206 3359 1455 
53299 
Case #10:
75799 34257 14031 85386 95129 6787 76097 71680 52241 64784 
6551 34257 14031 42965 59813 79687 37596 76097 76330 80779 68085 
